ui/src/components/yaml/styles.module.less

```diff
@@ -324,17 +324,14 @@
       }
     }
 
-    .yaml_content_diagnosisContent {
+    .yaml_content_diagnosisBody {
+      position: relative;
       width: 400px;
```

Comment:
Removing `color`, `background`, `padding`, `font-size`, and `line-height` properties may impact text appearance and readability.